entrypoint.sh

```diff
@@ -19,7 +19,8 @@ if [ ! -d "/github/workspace/${INPUT_WIKI_FOLDER}" ]; then
     exit 1
 fi
 cp -a ${INPUT_WIKI_FOLDER}/. /wiki
-
+chown -R $USER:$USER /wiki
+git config --global --add safe.directory /wiki
```

Comment:
As this has security implications, I'd suggest this is instead an optional component.

I would instead suggest that a new optional input `ignore_safe_warnings` is added that defaults to false, thus maintaining the same conditions for any users. This `chown` and `git config` command can be moved into an IF statement that only runs if `ignore_safe_warnings` is true.